content/en/05-reference/01-admin/09-user-mgmt.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "User Management & Sharing"
+metaTitle: "Reference | Admin | User Management & Sharing"
+metaDescription: "Share and manage access to your Pixie data."
+order: 6
+redirect_from:
+    - /admin/permissions/
+---
+
+<Alert variant="outlined" severity="info">This information applies to Pixie instances that use Auth0 for Authentication. <a href="https://docs.px.dev/installing-pixie/install-guides/community-cloud-for-pixie/">Community Cloud for Pixie</a> ships with Auth0. <a href="https://docs.px.dev/installing-pixie/install-guides/self-hosted-pixie/">Self-Hosted Pixie</a> must be <a href="https://docs.px.dev/reference/admin/authentication/#enabling-auth0">manually configured</a> to use Auth0. To learn how to invite users to Self-Hosted Pixie instances using open source authentication, see the Quick Start <a href="https://docs.px.dev/installing-pixie/install-guides/self-hosted-pixie/#invite-others-to-your-organization-(optional)">instructions</a>.
+</Alert>
+
+Pixie has features to share and manage access your Pixie organization:
+
+- [Invite users](/reference/admin/user-mgmt#invite-users)
+- [Enable approvals for new users](/reference/admin/user-mgmt#enable-approvals-for-new-users)
+- [Disable existing invite links](/reference/admin/user-mgmt#disable-invite-links)
+- [Remove users](/reference/admin/user-mgmt#remove-users)
+
+Users added to your organization can view Pixie live views, query running clusters, and deploy new Pixie clusters.
+
+## Invite Users
+
+### GSuite Organizations
+
+Pixie handles GSuite orgs differently than other orgs.
+
+You are part of a Pixie GSuite org if:
+
+1. The original creator of the org signs into Pixie using the "Sign in with Google" button AND
+2. The email the admin uses to sign in is part of a GSuite account.
+
+GSuite orgs can invite two types of users:
+
+- **Teammates with GSuite emails**: should use the "Sign-Up with Google" button on the [Sign up](https://work.withpixie.ai/auth/signup) page. GSuite emails will automatically join the Pixie org corresponding to their GSuite domain. _DO NOT use Invite Links to invite teammates with GSuite emails._ For greater control, enable [User Approvals](/reference/admin/user-mgmt#enable-approvals-for-new-users).
```

Comment:
> DO NOT use Invite Links to invite teammates with GSuite emails.

@aimichelle can you weigh in on this - it seems like it will be an easy mistake for users to make. I wonder if it makes sense to augment the flow to prevent people from falling into pits.

Reply:
This will already gate users and return an error message of the form 
`gsuite users are not allowed to follow invites. Please join the org with another account`
We can improve the error and add some more safeguards on our side as need be.

cc @philkuz

Reply:
A quick improvement we could do is to check if the GSuite domain matches the domain for the invite link they followed and let the user continue in that scenario (because that's the org the user would be in without and invite if they just did Google SSO).